resdk/resources/collection.py

```diff
@@ -31,7 +31,7 @@ class BaseCollection(BaseResolweResource):
         'description', 'descriptor', 'descriptor_schema', 'settings', 'tags',
     )
 
-    ALL_PERMISSIONS = ['view', 'download', 'add', 'edit', 'share', 'owner']
+    ALL_PERMISSIONS = ['view', 'edit', 'share', 'owner']
```

Comment:
Could we define those on `BaseResolweResource` or make it constant?

Reply:
Defined in `BaseResolweResource`.